.github/workflows/php.yml

```diff
@@ -0,0 +1,52 @@
+name: PHP Static Analysis & Tests
+
+on:
+  push:
+    branches: [ develop, 'issue/592' ]
```

Comment:
Is this on purpose or do you want to change this? :D

Also: Why on `develop`? Shouldn't this only be on the `master` branch?

Reply:
This is on purpose because the pushes are on the issue-specific branch, it should be dropped after merging.

Any actively-maintained branch should be included (i.e. possibly including the 3.x branch?), I just used develop because that's the one the branch was forked from.